compliance/components/ManualProcedures/component.yaml

```diff
@@ -44,32 +44,24 @@ satisfies:
   implementation_status: complete
   narrative:
     - text: |
-        Because the infrastructure is all code, Disaster Recovery into another region
-        should be simple, only requiring a change to the terraform code to change the
-        region and a restore of a database backup from the old region.
+        Because the infrastructure is all code, Disaster Recovery
+        should be simple, only requiring a relaunch and a restore of a database backup
```

Comment:
Seems like we might want to replace the "should be simple" with "Execute the [Deploy the Cloud.gov Script](https://github.com/HHS/TANF-app/blob/master/deploy-cloudgov.sh)" And some initial steps about how the db backup would be addressed.